spec/jekyll-feed_spec.rb

```diff
@@ -411,11 +411,9 @@ def to_s
       end
 
       it "outputs the category feed" do
-        expect(news_feed).to match '<title type="html">My awesome site | News</title>'
-        expect(news_feed).to match "http://example.org/news/2014/03/02/march-the-second.html"
-        expect(news_feed).to match "http://example.org/news/2013/12/12/dec-the-second.html"
-        expect(news_feed).to_not match "http://example.org/updates/jekyll/2014/03/04/march-the-fourth.html"
-        expect(news_feed).to_not match "http://example.org/2015/08/08/stuck-in-the-middle.html"
+        expect(category_feed).to match '<title type="html">My awesome site | News</title>'
+        expect(category_feed).to match "http://example.org/updates/jekyll/2014/03/04/march-the-fourth.html"
+        expect(category_feed).to match "March the fourth!"
```

Comment:
Could we cover the `category: jekyll` case too? The fixtures don't seem to have a post with the singular key name

Reply:
> The fixtures don't seem to have a post with the singular key name

The existing test on `master` uses a fixture post with a singular key name..